doc/source/whatsnew/v0.16.1.txt

```diff
@@ -16,8 +16,41 @@ We recommend that all users upgrade to this version.
 
 Enhancements
 ~~~~~~~~~~~~
+.. _whatsnew_0161.enhancements.sample:
 
+Sample
+^^^^^^^^^^^^^^^^
```

Comment:
The number of characters in this line should match the length of the preceding line, e.g., you only need `^^^^^^`